.changeset/olive-crabs-vanish.md

```
@@ -0,0 +1,7 @@
+---
+'@mastra/github-signals': minor
+---
+
+Added multi-PR GitHub signal tools and improved notification filtering.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)